packages/glimmer-runtime/tests/updating-test.ts

```diff
@@ -134,6 +134,31 @@ test("updating a single trusting curly", () => {
   notStrictEqual(root.firstChild.firstChild.firstChild, valueNode, "The text node was blown away");
 });
 
+test("updating a single trusting curly with siblings", function() {
+  let value = '<b>brave new </b>';
+  let context = { value };
+  let getDiv = () => root.firstChild;
+  let template = compile('<div>hello {{{value}}}world</div>');
+  render(template, context);
+
+  equalTokens(root, '<div>hello <b>brave new </b>world</div>', 'Initial render');
+
+  rerender();
+
+  equalTokens(root, '<div>hello <b>brave new </b>world</div>', 'rerender');
+
+  context.value = 'another ';
+  rerender();
+
+  equal(getDiv().firstChild.textContent, 'hello ');
+  equalTokens(getDiv().childNodes[1].textContent, 'another ');
+  equal(getDiv().lastChild.textContent, 'world');
+
+  rerender({value});
+
+  equalTokens(root, '<div>hello <b>brave new </b>world</div>', 'rerender');
+});
```

Comment:
This is a good test (and we should keep it). However, I don't think it would have caught the specific bug you fixed: the bug requires `nextSibling` to be `null`, which `{{{value}}}` has to be the last child of the parent, something like `<div>hello {{{value}}}</div>`.

Maybe we should just test all three sibling cases (via duplication is fine, but if you want to abstract it :+1:):

* `<div>{{{value}}} world</div>` (tests `prev = null`)
* `<div>hello {{{value}}} world</div>` (the one you have here)
* `<div>hello {{{value}}}</div>` (tests `nextSibling = null`)